platform/android/export/gradle_export_util.cpp

```diff
@@ -283,6 +283,7 @@ String _get_application_tag(const Ref<EditorExportPreset> &p_preset, bool p_has_
 		}
 	} else {
 		manifest_application_text += "        <meta-data tools:node=\"remove\" android:name=\"com.oculus.supportedDevices\" />\n";
```

Comment:
We'll need @kisg or @konczg input here but I think this should be removed here completely. The manifest additions should be made through the AAR loader plugin.

Reply:
If we don't add it outside of the plugin, we do not need it. I agree that having all manifest changes in the android plugin is cleaner, I just wanted to keep Pico and Quest consistent in this PR.

I assume / hope that the plan is to also move Quest things? That would avoid Quest metadata in Pico builds (which is not an issue, but also not really clean imho).

Reply:
Yup, that is the plan as far as I know, I think most of this code is left over from how things worked in Godot 3

Reply:
We discussed this with @kisg and for our current purposes, we also think these should be moved to the manifest of the specific AAR loader, which will be merged with the application's manifest. This might not cover all scenarios (e.g. if the permission/feature/meta-data is dependent on an export setting), for which the plugins might need to extend the manifest generation, but we do not have a proposal ready for that yet.

Reply:
Yeah like setting the hand tracking settings or passthrough settings, I can see that those would require enhancements to the export system where AARs could somehow add new export settings and embed some logic in the editor.

I guess that would be accomplished with export plugins, there should be some way to embed an export plugin into an AAR which the editor then loads, but which does not become part of what is deployed. (https://docs.godotengine.org/en/stable/classes/class_editorexportplugin.html)

That falls outside of the scope of this PR though.

Reply:
As discussed, this part of the patch is no longer part of this PR and now handled in https://github.com/GodotVR/godot_openxr_loaders/pull/13